tasks/ubuntu/bionic.task/metadata.yaml

```diff
@@ -0,0 +1,6 @@
+---
+
+os_version: bionic
+label: Ubuntu Bionic 18.04
+description: Ubuntu Bionic Installer
+base: ubuntu
```

Comment:
It looks like `ubuntu/bionic.task` is missing the README (should be same as `ubuntu.task/README.md` in this case).